paddle_ocr/main.py

```diff
@@ -1,18 +1,63 @@
-from absl import app
-from absl import flags
-from paddleocr import PaddleOCR
+from typing import Annotated, Any
+
 import paddle
+import pykka
+import uvicorn
+from absl import app, flags
+from fastapi import FastAPI, File
+from fastapi.responses import JSONResponse
+from paddleocr import PaddleOCR
+
+_MOCK = flags.DEFINE_boolean(
+    "mock",
+    False,
+    "Skips inference and outputs constant mock example with consistent structure.",
+)
+
 
-_IMAGE_URI = flags.DEFINE_string("image_uri", None, "URI of the image to process.")
+class Model(pykka.ThreadingActor):
+    _MOCK_RES = [
+        [
+            [[441.0, 174.0], [1166.0, 176.0], [1165.0, 222.0], [441.0, 221.0]],
+            ("ACKNOWLEDGEMENTS", 0.9971134662628174),
```

Comment:
Lol what is this

Reply:
Sample output so I can test it on macOS without a segfault. The schema is the same as actual output though I think